lib/commons/text/accessible-text-virtual.js

```diff
@@ -56,7 +56,7 @@ text.accessibleTextVirtual = function accessibleTextVirtual(
 		return step(virtualNode, context);
 	}, '');
 
-	if (context.startNode === virtualNode) {
+	if (context.sanitize && context.startNode === virtualNode) {
```

Comment:
Strong preference for using `context.unsanitized`. That way all truthy values do one thing, and all falsey values do another, as opposed to now where undefined behaves as though it's truthy.